src/main/java/UnknownCommandException.java

```diff
@@ -0,0 +1,11 @@
+public class UnknownCommandException extends DukeException{
+
+    public UnknownCommandException() {
+        super("OOPS!!! I'm sorry, but I don't know what that means :-(\n");
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Perhaps this ```toString()``` method could be removed for the exception classes? The message could be retrieved with the built-in method of the Exception class buy using ```getMessage()``` function.